tutorials/30_File_Type_Preprocessing_Index_Pipeline.ipynb

```diff
@@ -0,0 +1,1328 @@
+{
```

Comment:
<p>We havent provided the mime type for markdown files as it wasn't working. Since the <a href="https://github.com/deepset-ai/haystack/issues/6391" target="_blank">relevant issue</a> is resolved, we can add <code>text/markdown</code> to the <code>mime_types</code> list </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-tutorials/pull/274/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>